src/awkward/operations/ak_to_cudf.py

```diff
@@ -18,4 +18,4 @@ def to_cudf(
     """
     import cudf
 
-    return cudf.Series(array.layout._to_cudf(cudf, None, len(array)))
+    return cudf.Series._from_column(array.layout._to_cudf(cudf, None, len(array)))
```

Comment:
This functionality has become private. Is it something we're not supposed to do anymore?

Reply:
It would be worth finding the git blame on this, and asking on their repo